### ISSUE TITLE
add setting to collapse  outline view  

### ISSUE THREAD

**Issue by adil-s** [microsoft/vscode#107273]
currently, whenever a file is open in vs code editor, all nodes of outline view tree are expanded which is not screen reader friendly.
I request you to add setting to control this behaviour. so, whenever I open the file in VS code,  outline view tree nodes are collapse.
there is another little annoying behaviour for screen reader as well. 
when arrowing in outline tree view, NVDA keeps saying, "not selected node name". but it kinds of annoys me. as I know, outline view doesn't have any special meaning for selection. I.E. you  could delete the selected. 
 

@isidorn

**Comment by isidorn** [comment]
Assigning to @jrieken for the setting to contorl the expansion of the outline.
I would personally think that a command would be better such that the user can assign a keybdining to it and hopefully we preserve the expansion state

As for the "not selected" that is a tree behavior. We had a lot of discussions here https://github.com/microsoft/vscode/issues/91061

**Comment by jrieken** [comment]
This is a duplicate. Unfortunately, I don't recall the exact issue number nor its title :flushed: Anyways, searching for issues with the same labels will likely help you :point_right:
https://github.com/microsoft/vscode/issues?utf8=✓&q=is%3Aopen+label%3Afeature-request+label%3Aoutline

**Comment by isidorn** [comment]
Here's the issue for reference https://github.com/microsoft/vscode/issues/53262
